docs/resources/aws_db_subnet_group.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Please add links for the newly created resources' docs files in the project's README.

Reply:
Updated in recent commit